packages/devtools_app/lib/src/device_dialog.dart

```diff
@@ -78,11 +86,22 @@ class DeviceDialog extends StatelessWidget {
         ],
       ),
       actions: [
+        _connectToNewAppButton(context),
         if (connectedApp.isRunningOnDartVM) _ViewVMFlagsButton(),
         DialogCloseButton(),
       ],
     );
   }
+
+  Widget _connectToNewAppButton(BuildContext context) {
+    return RaisedButton(
+      onPressed: () {
+        DevToolsRouterDelegate.of(context).navigate(homePageId, {'uri': null});
+        Navigator.of(context, rootNavigator: true).pop('dialog');
+      },
+      child: const Text('CONNECT TO A NEW APP'),
```

Comment:
For consistency should this be `toUpperCase` as well?

Reply:
done